pontoon/base/models.py

```diff
@@ -2147,6 +2163,9 @@ def asymmetric(self):
 class Resource(models.Model):
     project = models.ForeignKey(Project, models.CASCADE, related_name="resources")
     path = models.TextField()  # Path to localization file
+    order = models.PositiveIntegerField(
+        default=0
+    )  # Index in the alphabetically sorted list of resources
```

Comment:
As a docstring, this can show up in code completion.

```suggestion
    )
    """
    Index in the alphabetically sorted list of resources

    Sorting resources by path is a heavy operation, so we use this field
    to represent the alphabetic order of resources in the project.
    """
```